demo/budg/dashboard/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@eubfr/demo-client",
+  "name": "@eubfr/demo-budg-dashboard",
```

Comment:
Could you please also update the target bucket from `bucketName: eubfr-${self:provider.stage}-site` to `bucketName: eubfr-${self:provider.stage}-demo-dashboard` to be found easier in S3?